test/actions/katello/repository/metadata_generate_test.rb

```diff
@@ -72,5 +72,17 @@ module Actions
       assert_action_planed_with(action, pulp_publish_class, yum_repo, SmartProxy.pulp_master,
                                 yum_action_options)
     end
+
+    it 'plans a yum refresh with matching content true, but during repository creation mode' do
```

Comment:
This is a valid test case but I don't think it's the one described in the fix (it passes when I revert your code change). Should there be a test to assert that a publish action wasn't planned with the `not_falsey `value or something similar?

Reply:
Good eye :+1:

Reply:
updated and added a 2nd test